psalm.xml.dist

```diff
@@ -37,6 +38,18 @@
         <file name="Registry.php"/>
     </projectFiles>
     <issueHandlers>
+        <InternalClass>
+            <errorLevel type="suppress">
+                <!-- Random package should not call this class from the Doctrine bridge, but we are in the Doctrine bundle here -->
+                <referencedClass name="Symfony\Bridge\Doctrine\Middleware\Debug\Driver"/>
+            </errorLevel>
+        </InternalClass>
+        <InternalMethod>
+            <errorLevel type="suppress">
+                <!-- Random package should not call this method from the Doctrine bridge, but we are in the Doctrine bundle here -->
+                <referencedMethod name="Symfony\Bridge\Doctrine\Middleware\Debug\Driver::__construct"/>
+            </errorLevel>
+        </InternalMethod>
```

Comment:
do these not work as `@psalm-suppress`?

Reply:
We do have a lot `psalm-suppres` in the code and we do have a lot `suppress` in the psalm config. Is there a pattern in DoctrineBundle when to use one or the other?

Reply:
Pattern is that we put stuff into config that does not work with inline suppress. There are plenty of such cases when psalm-suppress doesn't work but putting it to config does.

Reply:
> do these not work as `@psalm-suppress`?

I'll try, it's just that I forgot that we were doing things like this in that repository.

Reply:
It works.